tests/package/test_scp.py

```
@@ -0,0 +1,414 @@
+    proj = Project("testproj")
+    proj.option.set_cachedir(tmp_path)
+
+    resolver = SCPResolver("testscp", proj, "scp://github.com/test_owner/test_repo", "v1.0")
+    mock_which.return_value = "/usr/bin/scp"
+    mock_run.return_value = MagicMock(returncode=0)
+
+    resolver.resolve_remote()
+
+    mock_which.assert_called_once_with("scp")
+    mock_run.assert_called_once()
+    args, kwargs = mock_run.call_args
```

Ruff (0.15.9)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)